jekyll/add-ons.markdown

```diff
@@ -302,32 +302,6 @@ class MyIndexingEnhancement < RubyIndexer::Enhancement
 end
 ```
 
-Finally, we need to register our enhancement in the index once during the add-on's activation.
-
-```ruby
-module RubyLsp
-  module MyLibrary
-    class Addon < ::RubyLsp::Addon
-      def activate(global_state, message_queue)
-        # Register the enhancement as part of the indexing process
-        global_state.index.register_enhancement(MyIndexingEnhancement.new(global_state.index))
```

Comment:
`register_enhancement` was removed in https://github.com/Shopify/ruby-lsp/pull/2857